src/test/java/io/redis/examples/CmdsGenericExample.java

```diff
@@ -0,0 +1,435 @@
+// EXAMPLE: cmds_generic
+// REMOVE_START
+package io.redis.examples;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+// REMOVE_END
+// HIDE_START
+import redis.clients.jedis.UnifiedJedis;
+import redis.clients.jedis.args.ExpiryOption;
+// HIDE_END
+
+// HIDE_START
+public class CmdsGenericExample {
+    @Test
+    public void run() {
+        UnifiedJedis jedis = new UnifiedJedis("redis://localhost:6379");
+
+        //REMOVE_START
+        // Clear any keys here before using them in tests.
+
+        //REMOVE_END
+// HIDE_END
+
+
+        // STEP_START copy
+
+        // STEP_END
+
+        // Tests for 'copy' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START del
+        String delResult1 = jedis.set("key1", "Hello");
+        System.out.println(delResult1); // >>> OK
+
+        String delResult2 = jedis.set("key2", "World");
+        System.out.println(delResult2); // >>> OK
+
+        long delResult3 = jedis.del("key1", "key2", "key3");
+        System.out.println(delResult3); // >>> 2
+        // STEP_END
+
+        // Tests for 'del' step.
+        // REMOVE_START
+        Assert.assertEquals("OK", delResult1);
+        Assert.assertEquals("OK", delResult2);
+        Assert.assertEquals(2, delResult3);
+        // REMOVE_END
+
+
+        // STEP_START dump
+
+        // STEP_END
+
+        // Tests for 'dump' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START exists
+
+        // STEP_END
+
+        // Tests for 'exists' step.
+        // REMOVE_START
+
+        // REMOVE_END
+
+
+        // STEP_START expire
+        String expireResult1 = jedis.set("mykey", "Hello");
+        System.out.println(expireResult1);  // >>> OK
+
+        long expireResult2 = jedis.expire("mykey", 10);
+        System.out.println(expireResult2);  // >>> 1
+
+        long expireResult3 = jedis.ttl("mykey");
+        System.out.println(expireResult3);  // >>> 10
```

Comment:
time specific

Reply:
@sazzad16 Hi. Unless it causes problems for the testing system, I'm happy just to leave them like this (I figure that it will usually take less than half a second to run the tests anyway). There's no "correct" way to document what the user will see but providing some reasonable sample output helps to describe what happens.

Reply:
Could you use something like below? https://github.com/redis/jedis/blob/4c094726afe782d82fce000660efe78ea967a045/src/test/java/redis/clients/jedis/commands/unified/HashesCommandsTestBase.java#L641